### PR TITLE
Add instructions for manual testing in DG

### PR DESCRIPTION
**Pull request by brein62**
For the features:
- Material tracking
- Roles tracking
- Maximum volunteer count

Use cases:
- Update use cases for eaddv to include role and material tracking
- Add use case for eaddm

Suggestions for how to simplify these features are welcome

### CODE REVIEW

File: docs/DeveloperGuide.md
Comment (zacwong2151) [inline docs/DeveloperGuide.md:836]: should change `substitute X with the` to `where X is the`

File: docs/DeveloperGuide.md
Comment (zacwong2151) [inline docs/DeveloperGuide.md:846]: damn ok I just learnt about this LOL

**Review by zacwong2151 — APPROVED** [review]
lengthy test cases but I think its fair

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:957]: good idea cos i did this for the subsequent ones and forgot change hahah

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:967]: yessir the more you know

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md]: fixed :D

**Review by tiongMax — COMMENTED** [review]
Lmao i did the use cases for eaddv, eremovev, elistv ady, such a coincidence

**Comment by tiongMax** [comment]
> Lmao i did the use cases for eaddv, eremovev, elistv ady, such a coincidence

**Comment by tiongMax** [comment]
Oops accidentally closed it, its reopened now. My bad !

**Review by tiongMax — APPROVED** [review]
Lgtm!

File: docs/DeveloperGuide.md
Comment (ruo-x) [inline docs/DeveloperGuide.md:1118]: shouldnt this be under eshow?

**Review by ruo-x — CHANGES_REQUESTED** [review]
v detailed instructions but dk if the last test case considered part of eshow

File: docs/DeveloperGuide.md
Comment (jordankanghm) [inline docs/DeveloperGuide.md:791]: shld this be "iVolunter adds the volunteer to..." to simulate user and system interaction?

File: docs/DeveloperGuide.md
Comment (jordankanghm) [inline docs/DeveloperGuide.md:830]: same here, maybe can include iVolunteer in some manner?

File: docs/DeveloperGuide.md
Comment (jordankanghm) [inline docs/DeveloperGuide.md:878]: events!

**Review by jordankanghm — CHANGES_REQUESTED** [review]
just some minor comments but lgtm overall!

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:1118]: this reflects the `eshow` feature capability to show max volunteers. u can add to ur `eshow` test cases then i'll adjust the test case accordingly

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:791]: will fix

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:878]: i like to list all event too bad

File: docs/DeveloperGuide.md
Comment (brein62) [reply docs/DeveloperGuide.md:1118]: hmm i saw your `eshow` implementation and i think it's fine as long as it's listed after `eshow`. else the same issue would apply for `ecreate` etc.

**Comment by tiongMax** [comment]
Yo guys pls merge this after #229 is merged

File: docs/DeveloperGuide.md
Comment (ruo-x) [reply docs/DeveloperGuide.md:1118]: ok suree we can leave it as it is bah :)

Comment (brein62) [reply docs/DeveloperGuide.md:1118]: yay i'm so happy :D :D :D :D :D :D :D :D

Comment (ruo-x) [reply docs/DeveloperGuide.md:1118]: ?? JAJAHAHA

**Review by ruo-x — APPROVED** [review]
lgtm!!

**Pull request merged**
